stories/Components/UIcomponents/Hero/PageHero/PageHeroOption.jsx

```diff
@@ -3,36 +3,51 @@ import './page-hero.scss';
 import { Video } from '../../../../Atom/Video/Video';
 import { Heading } from '../../../../Atom/Typography/Heading/Heading';
 import { Breadcrumbcomponent } from '../../../Navigationcomponents/Breadcrumbs/Breadcrumbs';
+import { CtaButton } from '../../Buttons/CtaButton/CtaButton';
 
-export const PageHeroOption = ({
-  data, title, content, active, variant, imgsrc, imgsrc2, imgalt, videosrc,
-}) => (
-  <div className="pagehero-tall">
-    <div className="grid-x">
-      <div className="cell medium-4 medium-offset-1">
-        <div className="pagehero-content">
-          <Breadcrumbcomponent data={data} />
-          {content && <Heading type="4" label={content} dataViewport="true"/>}
-          <Heading type="2" label={title} dataViewport="true"/>
+export function PageHeroOption({
+  data,
+  title,
+  subtitle,
+  content,
+  active,
+  variant,
+  imgsrc,
+  imgsrc2,
+  imgalt,
+  videosrc,
+  cta,
+  ...args
+}) {
+  const heroImageClasses = variant === 'no_background' ? 'hero-placeholder' : 'homepage-hero-image';
```

Comment:
Can you avoid using specific classes for page hero variations? ideally implementation should use the same markup for all three, just styled appropriately if `<video>` is provided, or `<image>` is used or there is no media. 
The goal here is to provide flexible framework with minimal overhead.

Reply:
This is removed as the PageHeroOption doesn't need the no image variant. Only the main PageHero component has the no background option and a special class is no longer needed for it.